internal/db/schema/migrations/oss/postgres/99/01_worker_controller_tables.up.sql

```diff
@@ -71,37 +71,32 @@ drop table server_tag;
 -- Update session table to use worker_id instead of server_id, drop view first because of dependency on server type
 drop view session_list;
 
+-- Update session table to use worker_id instead of server_id
+-- Updating the session table modified in 01/01_server_tags_migrations.up.sql
 alter table session
   drop constraint session_server_id_fkey;
 drop trigger update_version_column
   on session;
 alter table session
   drop column server_type;
-
-alter table session
-  rename column server_id to worker_id;
 alter table session
-  add constraint server_worker_fkey
-    foreign key (worker_id)
-      references server_worker(private_id)
-      on delete set null
-      on update cascade;
+  drop column server_id;
 
 create trigger
   update_version_column
-  after update of version, termination_reason, key_id, tofu_token, worker_id on session
+  after update of version, termination_reason, key_id, tofu_token on session
   for each row execute procedure update_version_column();
 
 -- Update session_connection table to use worker_id instead of server_id
+-- Table last updated in 21/02_session.up.sql
 alter table session_connection
-  drop constraint server_fkey;
-
+  drop column server_id;
 alter table session_connection
-  rename column server_id to worker_id;
+  add column worker_id wt_public_id;
 alter table session_connection
   add constraint server_worker_fkey
     foreign key (worker_id)
-      references server_worker (private_id)
+      references server_worker (public_id)
       on delete set null
       on update cascade;
```

Comment:
Same question for the `session_connection` table and the `job_run` table:

```sql
alter table session_connection
  drop column server_id,
  add column worker_id wt_public_id
    constraint server_worker_fkey
      references server_worker (public_id)
      on delete set null
      on update cascade;

-- Update job run table so that server id references controller id
alter table job_run
  drop constraint server_fkey,
  add constraint server_controller_fkey
    foreign key (server_id)
      references server_controller (private_id)
      on delete set null
      on update cascade;
```